include/DREAM/Equations/EffectiveCriticalField.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef _DREAM_EQUATIONS_EFFECTIVE_CRITICAL_FIELD_HPP
+#define _DREAM_EQUATIONS_EFFECTIVE_CRITICAL_FIELD_HPP
+
+namespace DREAM { class EffectiveCriticalField; }
+
+
+#include "DREAM/Equations/EffectiveCriticalField.hpp"
+#include "DREAM/Equations/AnalyticDistribution.hpp"
+#include "DREAM/Equations/PitchScatterFrequency.hpp"
+#include "DREAM/Equations/RunawayFluid.hpp"
+#include "DREAM/Equations/SlowingDownFrequency.hpp"
+#include "DREAM/IonHandler.hpp"
+
+
+//#include "FVM/config.h"
+
+namespace DREAM {
+    class EffectiveCriticalField {
+    public:
+        struct ParametersForEceff {FVM::RadialGrid *rGrid; SlowingDownFrequency *nuS; PitchScatterFrequency *nuD; FVM::fluxGridType fgType; 
+                            gsl_integration_workspace *gsl_ad_w;
+                            gsl_min_fminimizer *fmin; CollisionQuantity::collqty_settings *collSettingsForEc;
+                            CollisionQuantity::collqty_settings *collQtySettings; gsl_root_fsolver *fsolve; 
+                            OptionConstants::collqty_Eceff_mode Eceff_mode;
+                            IonHandler *ions;
+                            CoulombLogarithm *lnLambda;};
```

Comment:
I think this struct could be declared a bit more nicely, with one type per line (and the ``{`` ending its line). You can also declare ``collSettingsForEc`` and ``collQtySettings`` on the same line using
```
CollisionQuantity::collqty_settings *collSettingsForEc, *collQtySettings;
```